lstchain/data/lstchain_standard_config.json

```diff
@@ -65,6 +65,10 @@
     "fraction_cleaning_intensity": 0.03
   },
 
+  "random_forest_weight_settings": {
+    "pointing_wise_weights": true
+  },
```

Comment:
> is it expected to have more settings in `random_forest_weight_settings`? Otherwise, it may be simplified to `"random_forests _pointing_weights": true`

I thought we might want to set this independently for each RF - I tend to think otherwise now

Reply:
Still, I think this way allows better future enhancements (like class weights) without changing the current way of setting this.